README.md

```diff
@@ -3,40 +3,58 @@
 
 TripleA is a free game engine that runs on open source and is community supported.
 
-Installing TripleA and Playing
-==============================
-- Download and install TripleA: http://triplea-game.github.io/download/
-
-TripleA Websites
-================
-- HomePage: http://www.triplea-game.org/
-- Community Website: http://www.tripleawarclub.org/
-- Community Maps Repository: http://github.com/triplea-maps
+# Installing TripleA and Playing
+
+- Download and install TripleA: http://triplea-game.org/download/
+
+# TripleA Websites
+
+- Home Page: http://triplea-game.org/
+- Community Website: https://forums.triplea-game.org/
+- Community Maps Repository: https://github.com/triplea-maps/
 - Source Forge (legacy): https://sourceforge.net/projects/triplea/
 
-TripleA Game Features
-=====================
+# TripleA Game Features
+
 - Free to play, 100% open source and community supported
 - Online lobby, find, join and observe live games
 - Play by correspondence (email)
 - Play single player against the AI
 - Many community created maps available for in-game download
 - Generic game engine allows you to [build your own maps and mods](https://github.com/triplea-maps/Project)
 
-Reporting Bugs 
-==============
+# Reporting Bugs
+
 - Please use: https://github.com/triplea-game/triplea/issues/new
 
-Feature Requests
-================
+# Feature Requests
+
 - Create a new forum topic at: https://forums.triplea-game.org/category/42/feature-requests-and-ideas
 
-Lobby Game Bot Hosting
-======================
-- Please see: https://github.com/triplea-game/lobby#run-hosted-bots
+# Lobby Game Bot Hosting
+
+- Please see: https://github.com/triplea-game/lobby#bots
+
+# Development
 
-Development
-===========
 - Project documentation, including 'how to get started guides' at: https://github.com/triplea-game/triplea/tree/master/docs/dev
 - Broken map list: https://github.com/triplea-game/triplea/wiki/Broken-Maps
 - Additional feature request list: https://github.com/triplea-game/triplea/issues?q=label%3A%22ice+box+-+revisit+later%22+is%3Aclosed (list is not to be added to, if picked up, re-open and remove the ice-box label)
```

Comment:
Unrelated, but the Ice Box label was renamed to `ice box - close and revisit later`, so this link is broken.
Additionally this link should be hidden behind a nicer name, it really does not look good in the mark down.

Reply:
Agreed.  In general, I don't like dropping bare URLs in Markdown, especially ones as busy as the ice box link.  How about if I just hide _all_ links in the README behind a label?  For example:

# Development

* [Project documentation](https://github.com/triplea-game/triplea/tree/master/docs/dev) (including 'how to get started guides')
* [Broken map list](https://github.com/triplea-game/triplea/wiki/Broken-Maps)
* [Additional feature request list](https://github.com/triplea-game/triplea/issues?q=label%3A%22ice+box+-+revisit+later%22+is%3Aclosed) (list is not to be added to, if picked up, re-open and remove the ice-box label)

Reply:
👍 For consistency

Reply:
0752369aa

Thanks for catching the broken ice box link.  I tested all the others; not sure why I skipped that one. 🤷‍